pkg/version/version.go

```diff
@@ -104,36 +104,30 @@ func GetLatestVersion(uuid string, source string) (string, error) {
 	return version, nil
 }
 
-func parseVersion(version string) (string, error) {
+func parseVersion(version string) string {
 	if parts := strings.SplitN(version, "-", 2); len(parts) == 2 {
-		return parts[1], nil
+		return parts[1]
 	}
-	return "", fmt.Errorf("Unsupported version format: %s", version)
+	return version
 }
 
-func parseChannel(version string) (string, error) {
+func parseChannel(version string) string {
```

Comment:
this is probably but, but i don't see a case in the code where `parseChannel` should permit a missing channel string. we're either doing `parseChannel(Version)`, which is the locally built CLI Version, or `parseChannel(expectedVersion)`, which comes from online at `version.json`.